.github/stale.yaml

```diff
@@ -1,26 +0,0 @@
-# Number of days of inactivity before an issue becomes stale
```

Comment:
We're dropping the stale stuff?

Reply:
Is we are actively categorizing/closing issues, I do not think this is needed.